src/UI/ledger/LedgerLoginContainer/AddressRow/AddressRow.tsx

```diff
@@ -48,10 +50,15 @@ export const AddressRow = ({
         className
       )}
     >
-      <div className={styles.ledgerAddressTableBodyItemData}>
+      <div
+        className={`${styles.ledgerAddressTableBodyItemData} ${classNames({
```

Comment:
Just `disabled` as a class name isn't going to work, you need to write it as `[styles.disabled]: disabled`. Also, this could be written as following:

`classNames(styles.ledgerAddressTableBodyItemData, { [styles.disabled]: disabled })`